### DISCUSSION TITLE
MISP: How to integrate VirusTotal (or any free/paid threat Intel Solution) with MISP Platform

### DISCUSSION THREAD

**Discussion by Aim-track** [MISP/MISP#8994] [Q&A]
Hi Folks,
I need your help to integrate VirusTotal (or any free/paid threat Intel Solution) with MISP Platform. 
FYI: I've just setup MISP Platform from scratch.

Thanks, Aim

**Comment by Nicolas-Pellletier** [comment]
You can set it up via [expansion modules](https://github.com/MISP/misp-modules). enable this modules in settings (https://misp-server/servers/serverSettings/Plugin) and add the required element in order to properly communicate with the third-party threat intelligence api (API key, server_url, client-id, etc...).
Then go on any event that you would like to enrich (then on the left panel you can see enrichement)

**Reply by abduskif** [reply]
May you explain step-by-step please, I couldn't do it previous 7 days.
